package.json

```
@@ -27,7 +27,7 @@
     "test:affected": "yarn nx affected --target=test",
     "test-e2e": "yarn nx run-many --target=test-e2e",
     "test-int": "yarn nx run-many --target=test-int",
-    "postinstall": "husky && yarn build:lint && yarn harmonize:version && yarn update-yarn-sdks",
+    "postinstall": "husky && yarn build:lint && yarn harmonize:version && yarn update-yarn-sdks && cross-env YARN_ENABLE_SCRIPTS=false yarn install",
```

```suggestion
    "postinstall": "husky && yarn build:lint && yarn harmonize:version && yarn update-yarn-sdks && yarn install --mode=skip-build",
```
was not doing the job?

---

it seems to be doing a better job.